.github/workflows/continuous-integration.yml

```
@@ -33,40 +30,22 @@ jobs:
 
     steps:
 
-    - name: Free Disk Space (Ubuntu)
-      uses: jlumbroso/free-disk-space@v1.3.1
-      if: runner.os == 'Linux'
-      with:
-        # this might remove tools that are actually needed,
-        # if set to "true" but frees about 6 GB. Currently we don't seem to need
-        # this extra disk space
-        tool-cache: false
-
-        # we might turn some of those off to trade CI time for less space saving
-        # if this step turns out to be too slow
-        android: true
-        dotnet: true
-        haskell: true
-        large-packages: true
-        docker-images: true
-        swap-storage: true
-
     - name: Checking out the repository
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
 
-    - name: Installing Nix
-      uses: cachix/install-nix-action@V27
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-        extra_nix_config: |
-          experimental-features = nix-command flakes
-          accept-flake-config = true
+    - name: Install cachix
+      id: install_cachix
+      run: |
+        set -euo pipefail
+        cachix_path="$(nix build --print-out-paths --inputs-from . nixpkgs#cachix.out)"
```

Ah, you need to use different output...

```suggestion
        cachix_path="$(nix build --print-out-paths --inputs-from . nixpkgs#cachix.bin)"
```